Sources/Public/DotLottie/DotLottieHelpers.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  DotLottieHelpers.swift
+//  Lottie
+//
+//  Created by Evandro Hoffmann on 20/10/22.
+//
+
+import Foundation
+
+extension DotLottie {
+
+  /// A closure for an Animation download. The closure is passed `nil` if there was an error.
+  public typealias DotLottieDownloadClosure = (DotLottie?) -> Void
+
+  /// Returns the list of `DotLottieAnimation` in the file
+  public var dotLottieAnimations: [DotLottieAnimation] {
```

Comment:
Does this need to be public? It seems like it's only used internally when loading the dotLottie archive contents in to memory